staging/src/k8s.io/kube-scheduler/framework/interface.go

```diff
@@ -622,6 +622,26 @@ type BindPlugin interface {
 	Bind(ctx context.Context, state CycleState, p *v1.Pod, nodeName string) *Status
 }
 
+type PodSignatureBuilder interface {
+	Unsignable()
+	AddPodElement(elementName string, obj any) error
+	AddSignatureVolumes(pod *v1.Pod) error
+	AddPluginElement(elementName string, obj any) error
+}
+
+const Unsignable = ""
+
+// BatchablePlugin is an interface that should be implemented by plugins that either filter or score
+// pods to enable batching and gang scheduling optimizations. If an enabled plugin that does Scoring,
+// Prescoring, Filtering or Prefiltering does not implement this interface we will turn off batching for all pods.
+type BatchablePlugin interface {
```

Comment:
Worth add a mention on Filter / Score plugin interface comment. Also, you need a release note because custom plugin devs likely want to implement this BatchablePlugin for their plugins. Worth letting them know on the release note.